napari_allencell_annotator/widgets/template_list.py

```diff
@@ -109,6 +110,11 @@ def add_item(self, name: str, key: Key | ComboKey):
                 for opt in key.get_options():
                     widget.addItem(opt)
             widget.setCurrentText(default)
+        else:
```

Comment:
elif

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/92/commits/6bfbf5e42452477f6bdbed152dcf5f16c8a90a38